### DISCUSSION TITLE
Gradle Build error

### DISCUSSION THREAD

**Discussion by soCallmeAdityaKumar** [oppia/oppia-android#4798] [Q&A (Installation)]
![image](https://user-images.githubusercontent.com/101629190/209130213-3a3a7408-9fc6-45e8-8c35-a3989d62ffd4.png)

Why this error is coming ?. I have proceeded according to the contributers guideline.

**Comment by BenHenning** [comment]
Can you try the build again @soCallmeAdityaKumar? Some of our artifacts are built using jitpack.io (including for spotlights), and so the first run can sometimes fail if there isn't a cached version to pull from jitpack.io. Trying the build again should work.

**Reply by soCallmeAdityaKumar** [reply]
No its still coming.(build several times)

**Comment by MohitGupta121** [comment]
@soCallmeAdityaKumar can you please try it by Build -> Clean Project -> Rebuild.
And which os and android studio version you using?

**Reply by soCallmeAdityaKumar** [reply]
I did the same process again and i don't know(how) it worked for me . Thanks . Excited for contributing

**Reply by MohitGupta121** [reply]
Thanks @soCallmeAdityaKumar feel free to ask any query in GitHub discussion.

If your problem solved please mark this discussion as resolved by marking the answer.
